build/tools/templates/Archive/Archive.csproj

```diff
@@ -3,9 +3,8 @@
   <Import Project="$(DotNetRestoreSourcePropsPath)" Condition="'$(DotNetRestoreSourcePropsPath)' != ''"/>
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
     <EnableApiCheck>false</EnableApiCheck>
-    <DotnetCliToolTargetFramework>netcoreapp2.1</DotnetCliToolTargetFramework>
```

Comment:
I'm assuming this is because tools are now global and we don't need them in the LZMA anymore?